pkg/server/apis/v1alpha1/types.go

```diff
@@ -345,3 +347,12 @@ const (
 	// CacheCleanupSucceeded ...
 	CacheCleanupSucceeded CacheCleanupPhase = "Succeeded"
 )
+
+// StageArtifact describes artifacts produced by a certain stage
+type StageArtifact struct {
+	// Stage name
+	Stage string `json:"stage"`
+	// Files name
+	File              string    `json:"file"`
```

Comment:
When there are many files, the stage name will be duplicated returned.
```suggestion
// StageArtifact describes artifacts produced by a certain stage
type StageArtifacts struct {
	// Stage name
	Stage string `json:"stage"`
	// Files name
	Files              []StageArtifactFile    `json:"files"`
}

type StageArtifactFile struct {
	// Files name
	File              string    `json:"file"`
	CreationTimestamp time.Time `json:"creationTimestamp"`
}
```

Reply:
The notes of `StageArtifact` is not correct, should be `StageArtifact describes artifacts produced by a workflowRun`